xds/internal/balancer/clusterresolver/e2e_test/eds_impl_test.go

```diff
@@ -340,7 +352,8 @@ func (s) TestEDS_MultipleLocalities(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	// Remove the first locality, and ensure roundrobin.
+	// Remove the first locality, and ensure RPCs are being weighted
+	// roundrobined across the last two backends.
```

Comment:
Nit: switch "last two backends" to "two backends remaining" or something.

Reply:
Done.